docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
```

Comment:
<img width="300" alt="ss3" src="https://user-images.githubusercontent.com/60163283/140517894-f3efc087-f455-4a1f-86db-72e3386deabb.PNG">
Should be self invocate getInstrumentAtIndex then getInstrument(1), can look at william's done feature ref diagram.